scripts/add-fields-scpca-meta.py

```diff
@@ -81,12 +87,24 @@
         print(f"No scpca-meta.json file for {run.scpca_run_id}")
         continue
 
-    # add mito file and ref fasta index if not already present
-    if all(key in results_meta for key in ('mito_file', 'ref_fasta_index')):
-        print(f"All keys are present, no updates to scpca-meta.json for {run.scpca_run_id}")
+    # create a list of new fields to check for
+    new_fields = ['mito_file', 'ref_fasta_index', 'assay_ontology_term_id', 'submitter_cell_types_file']
```

Comment:
For easier updates later, we might make this into a dict:
```suggestion
    new_fields = {
        'mito_file': "s3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.mitogenes.txt",
        'ref_fasta_index': "homo_sapiens/ensembl-104/fasta/Homo_sapiens.GRCh38.dna.primary_assembly.fa.fai",
        'assay_ontology_term_id', run.assay_ontology_term_id,
        'submitter_cell_types_file', run.submitter_cell_types_file
    }
```

We could then use a loop to set all of the `results_meta` values without all of the `if` statements:

```
for key, default in new_fields.items():
   results_meta.setdefault(key, default)
```

This uses `.setdefault()` which does nothing if the key exists, but updates the dictionary if it does. (Well, technically it also returns the value, but we don't need to capture that.)

This doesn't handle the case of updating the cell types file, but as that breaks the pattern, we might want to separate it more cleanly anyway.